src/FakeItEasy/CallbackConfigurationExtensions.StronglyTyped.tt

```diff
@@ -2,6 +2,14 @@
 <#@ output extension=".cs" #>
 <#@ assembly name="System.Core" #>
 <#@ import namespace="System.Linq" #>
+//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a T4 template.
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------

```

Comment:
Had to add this so that it's ignored by StyleCop. Previously the ".generated" in the file name was enough.
(I guess I *could* address the warnings, but it's a pain with T4)